### DISCUSSION TITLE
Are comments possible?

### DISCUSSION THREAD

**Discussion by mayagomez** [cnr-isti-vclab/TagLab#16] [General]
Is it possible to add a personalized comment to a shape in TagLab? For example, if a coral colony is bleached/diseased, is there a way to add that information for future monitoring? Thanks!

**Comment by maxcorsini** [comment]
Hi,

it is not possible to add a comment to a segmented region. Anyway, you can set up an annotation grid (using the last buttons of the toolbar). The annotation grid is typically used to track the annotation work, but it is also possible to add comments anywhere in the map.
If your purpose is annotate each coral this is not fit well but it is the only way to add personalized comments, for the moment. 

All the best
